Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Controller.purs

```diff
@@ -25,7 +25,7 @@ import Components.SourceToDestination as SourceToDestination
 import Data.Array ((!!), null, filter)
 import Data.Lens ((^.))
 import Data.Maybe (Maybe(..), fromMaybe)
-import Helpers.Utils (convertUTCtoISC)
+import Helpers.Utils (convertUTCtoISC, withinTimeRange)
```

Comment:
remove this

Reply:
Done, was testing something else. By mistake I added.